src/main/kotlin/car_racing/InputDto.kt

```diff
@@ -0,0 +1,37 @@
+package car_racing
+
+data class InputDto(val carCountString: String, val movementTryCountString: String) {
+    private val _carCount: UInt
+    val carCount: Int
+        get() = _carCount.toInt()
+    val _movementTryCount: UInt
+    val movementTryCount: Int
+        get() = _movementTryCount.toInt()
+
+    init {
+        validateForIntegerString(carCountString, CAR_COUNT_NAME)
+        validateForIntegerString(movementTryCountString, MOVEMENT_TRY_COUNT_NAME)
+
+        _carCount = removeSpace(carCountString).toUInt()
```

Comment:
내부적으로 UInt를 사용하는 곳이 없는데요! 바로 toInt로 변환해주면 어떨까요?

```suggestion
        carCount = removeSpace(carCountString).toInt()
```

Reply:
생각해 보니 앞에 `validation` 로직이 음수 값을 걸러내서 toUnit는 없어도 될것 같네요.
다시 `toInt`로 변경하겠습니다.
 
![image](https://user-images.githubusercontent.com/39614239/168458779-065729ef-40f3-4bac-98f8-faf708d8563a.png)